src/ui/components/Library/Team/View/Tests/Overview/TestErrorList.tsx

```diff
@@ -0,0 +1,176 @@
+import { motion } from "framer-motion";
+import orderBy from "lodash/orderBy";
+import { Dispatch, SetStateAction, useState } from "react";
+import { ContextMenuItem, useContextMenu } from "use-context-menu";
+
+import Icon from "replay-next/components/Icon";
+import { TestExecution } from "shared/test-suites/TestRun";
+
+import { getTruncatedRelativeDate } from "../../Recordings/RecordingListItem/RecordingListItem";
+import styles from "../../../../Library.module.css";
+import testPageStyles from "../TestsPage.module.css";
+
+const MAX_ERRORS_SHOWN = 5;
+const MAX_REPLAYS_SHOWN = 3;
+
+export function TestErrorList({
+  errorFrequency,
+  executions,
+}: {
+  errorFrequency: Record<string, number>;
+  executions: TestExecution[];
+}) {
+  const [selectedError, setSelectedError] = useState<string | null>(null);
+  const [filterByTime, setFilterByTime] = useState<number | null>(null);
+
+  const {
+    contextMenu: contextMenuTimeFilter,
+    onContextMenu: onClickTimeFilter,
+    onKeyDown: onKeyDownTimeFilter,
+  } = useContextMenu(
+    <>
+      <ContextMenuItem dataTestId="show-all-runs" onSelect={() => setFilterByTime(null)}>
+        All errors
+      </ContextMenuItem>
+    </>,
+    { alignTo: "auto-target" }
```

Comment:
Only adding one option to these menus makes them kind of useless:

<img width="174" alt="Screen Shot 2023-11-15 at 10 46 55 AM" src="https://github.com/replayio/devtools/assets/29597/24294838-1b44-48c6-bf5d-95afcdb95c9e">

<img width="106" alt="Screen Shot 2023-11-15 at 10 46 58 AM" src="https://github.com/replayio/devtools/assets/29597/4ffc9daf-7456-49df-9444-8faa887a74b7">

<img width="120" alt="Screen Shot 2023-11-15 at 10 47 02 AM" src="https://github.com/replayio/devtools/assets/29597/1c46da8c-13c6-45a6-a4d1-581f2016dd80">

There's no way to turn them off.

Reply:
Lets show the other options that are in the mock, but have them be disabled. This will let users see that they're coming